website/docs/cdktf/cli-reference/commands.mdx

```diff
@@ -676,3 +676,52 @@ Add a provider from a private registry to the project.
 ```bash
 $ cdktf provider add registry.example.com/acme/my-provider
 ```
+
+## provider add
+
+This command facilitates upgrading Terraform providers in a CDKTF project. If a [pre-built provider](/cdktf/concepts/providers#install-pre-built-providers) for the requested provider is installed we will update it to the newest version matching the version constraint and your `cdktf` version. Otherwise, the provider will be added to the `cdktf.json` config and `cdktf get` will be automatically invoked to generate [local provider bindings](/cdktf/concepts/providers#add-provider-to-cdktf-json).
```

Comment:
```suggestion
This command lets you upgrade Terraform providers in a CDKTF project. If a [pre-built provider](/cdktf/concepts/providers#install-pre-built-providers) for the requested provider is installed, CDKTF updates it to the newest version that matches the version constraint and your `cdktf` version. Otherwise, CDKTF adds the provider to the `cdktf.json` configuration file and invokes cdktf get` to generate [local provider bindings](/cdktf/concepts/providers#add-provider-to-cdktf-json).
```